CameraBrowser/CameraBrowserSample/ViewController.cs

```diff
@@ -0,0 +1,238 @@
+﻿using System;
+using AppKit;
+using Foundation;
+using ImageCaptureCore;
+using ObjCRuntime;
+#nullable enable
+
+namespace CameraBrowserSample
+{
+	public partial class ViewController : NSViewController, IICDeviceBrowserDelegate, IICCameraDeviceDelegate, IICCameraDeviceDownloadDelegate, IICDeviceDelegate {
+		public ICDeviceBrowser DeviceBrowser { get; set; } = new ICDeviceBrowser();
+		public NSMutableArray Cameras { [Export("Cameras")] get; [Export("setCameras:")] set; } = new NSMutableArray();
+
+		static ViewController() //init
+		{
+			var imageTransformer = new CGImageRefToNSImageTransformer();
+			NSValueTransformer.SetValueTransformer(imageTransformer, "CGImageRefToNSImage");
+		}
```

Comment:
This is just a nit but I wonder if this is the right place for this code, can't you have this done in the actual ctor that gets called?

You would have to look which of the initWithxxx selector is used to initialize the ViewController by ObjC and override that in C# and add that code there.